BankingKataSpecs/AccountSpecs/Features/CreateAccount.feature

```diff
@@ -0,0 +1,19 @@
+﻿Feature: Create account
+	In order to us my account
+	As a foo user
+	I want to be able to create an account.
+
+@mytag
+Scenario: New account has zero cash.
+	Given I want to create an account
+	When I create a new account
+	Then the account should have 0 cash
+
+Scenario: New account can't take null cash.
+	Given I want to create an account
+	And I have null cash
+	Then Creating an account should throw an argument null exception
```

Comment:
I agree with it feeling dev-orientated.  I think you can test failure conditions as user stories with BDD but you're right this particular example is a bad one and may be more suitable at a lower level.